src/http/get-ipr-000user-000repo-pull-000prEtc/index.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
+/* eslint no-magic-numbers: 0 */
+
 const path = require('path');
 const { execSync } = require('child_process');
-const GitHub = require('github-api');
+const { Octokit } = require('@octokit/rest');
 
-const gh = new GitHub();
+const octokit = new Octokit({});
 
-function html({
+const html = ({
```

Comment:
and this one?

Reply:
→ https://github.com/ljharb/tc39-ci/pull/27#discussion_r1592709066